apps/server/src/db/dictionary.ts

```diff
@@ -82,5 +84,10 @@ export const listByName = async (name: string): Promise<DictionaryDocument[]> =>
 export const addDictionary = async (dictionary: Dictionary): Promise<DictionaryDocument> => {
 	const dict = new DictionaryModel(dictionary);
 	const saved = await dict.save();
-	return saved.toObject();
+	return stripExtras(saved.toObject());
 };
+
+/**
+ * Remove the mongo version property __v from the response object
+ */
+const stripExtras = (doc: DictionaryDocument) => omit(doc, '__v');
```

Comment:
went through a rabbit hole trying to figure out whether `stripExtras` is a native js thing, or a TS thing, or what...

do we want to move this definition to somewhere before it's implemented/executed to prevent that happening to other devs?